src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![cfg_attr(not(test), no_std)]
-#![feature(asm)]
+#![feature(asm, global_asm, isa_attribute)]
```

Comment:
i don't see any global_asm in the PR, was this from a previous draft?

Reply:
Yeah. It's used in the save API change, and I missed this when I was splitting my changes between multiple PRs.